src/python/espressomd/lbboundaries.py

```diff
@@ -1,25 +1,27 @@
 from __future__ import print_function, absolute_import
 from .script_interface import ScriptInterfaceHelper
+import espressomd.code_info
 
 
-class LBBoundaries(ScriptInterfaceHelper):
-    _so_name = "LBBoundaries::LBBoundaries"
+if any(i in espressomd.code_info.features() for i in ["LB_BOUNDARIES", "LB_BOUNDARIES_GPU"]):
```

Comment:
The logic of your check is not quote right since you would have to have both features compiled in, if I'm getting it right. The check only fails because the system class does not check if the boundaries are compiled in.